app/views/users/index.html.erb

```diff
@@ -1,5 +1,22 @@
 <div class="users">
+  <% flash.each do |type, msg| %>
+    <div class="alert">
+      <% if type == 'error' then %>
+        <div class="alert_inner alert-danger">
+          <%= msg %>
+        </div>
+      <% end %>
+      <% if type == 'success' then %>
+        <div class="alert_inner alert-success">
+          <%= msg %>
+        </div>
+      <% end %>
+    </div>
+  <% end %>
   <h1>Users</h1>
+  <div class="create_post">
+    <%= link_to "Create Post", "/post/new" %>
+  </div>
   <% @users.each do |user| %>
     <div class="user_info_wrapper">
       <%= image_tag user.photo, :class => 'user_avatar' %>
```

Comment:
- [ ] Great job implementing the required changes from the previous reviewer. However, there're some misconceptions on the recommendations from the previous reviewer. This is how it should work.
- [ ] 1.  All the users added to the db with `rails console` should get displayed when the app starts. Each user name should be a link to the list of user's posts and a form to create new posts.
- [ ] Each post should be a link to the post show page where there would be a form to add comments to the posts and the posts could be liked.
- [ ] Kindly follow the wirefram from the project requirements to implement all this. You would not have any issues if you follow the wireframe properly.